src/Schema/Grammar.php

```diff
@@ -371,6 +371,7 @@ protected function addInterleaveToTable(Blueprint $blueprint)
     protected function addRowDeletionPolicy(Blueprint $blueprint)
     {
         if (! is_null($command = $this->getCommandByName($blueprint, 'rowDeletionPolicy'))) {
+            /** @var RowDeletionPolicyDefinition $command */
```

Comment:
should assert but that will break compatibility so it will be done in a later version